packages/gamut-labs/src/experimental/AppHeaderMobile/AppHeaderLinkMobile/__tests__/AppHeaderLinkMobile-test.tsx

```diff
@@ -0,0 +1,38 @@
+import { theme } from '@codecademy/gamut-styles';
+import { ThemeProvider } from '@emotion/react';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { AppHeaderLinkMobile, AppHeaderLinkMobileProps } from '..';
+
+const testText = 'Test Link';
+const onClick = jest.fn();
+
+const props: AppHeaderLinkMobileProps = {
+  item: {
+    dataTestId: '',
+    id: 'test-link',
+    href: 'test-url',
+    text: testText,
+    trackingTarget: '',
+    type: 'link',
+    icon: () => <svg />,
+  },
+  onClick: onClick,
```

Comment:
Josh gave me a tip last week that this line and similar ones can be changed to just `onClick`!
```suggestion
  onClick,
```

Reply:
fixed. thanks!